plugins/kotlin/idea/tests/testData/inspectionsLocal/redundantLambdaArrow/nullableReceiver.kt

```diff
@@ -0,0 +1,4 @@
+// PROBLEM: Redundant lambda arrow
+// WITH_STDLIB
+
+fun test(list: List<String>?) = list?.map { <caret>it -> it.length }.orEmpty()
```

Comment:
I would use Ivan's repro and remove orEmpty, it's much simpler. The orEmpty is not important to the issue, only the `it`.

https://youtrack.jetbrains.com/issue/KTIJ-20748#focus=Comments-27-5701887.0-0